constitution.tex

```diff
@@ -390,6 +391,16 @@
 	\item To ensure that there is a variety of activities for House members to participate in throughout the academic year
 \end{enumerate}
 
+\asubsection{Responsibilities of the Public Relations Director}
+\begin{enumerate}
+	\item To preside over Public Relations Directorship Meetings
+	\item To operate all House social media accounts
```

Comment:
The PR director is historically not responsible for certain accounts, notably OpComm twitter

Reply:
I can definitely see what you mean, but the way I interpret that is all social media accounts that represent house as a whole would be controlled by PR, and the power doesn't limit specific directors from creating their own accounts and running them.

Reply:
Being more clear, I wouldn't define OpComm twitter as a House account, but as an OpComm account only

Reply:
You should clarify that you mean `accounts that represent house as a whole`, since `House ... accounts` can read as `any and all accounts controlled by House`